website/docs/getting-started/docker-compose.md

```diff
@@ -0,0 +1,192 @@
+# Docker Compose
+
+The easiest way to run the ORT Server for testing is to use [Docker Compose](https://docs.docker.com/compose/).
+For a proper deployment to Kubernetes, the project will later provide a Helm chart.
+
+:::warning
+Do not use the Docker Compose setup in production as it uses multiple insecure defaults, like providing Keycloak without TLS.
+:::
+
+To use Docker Composer, you need to have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed.
+Also, you have to clone the ORT Server repository:
+
+```shell
+git clone https://github.com/eclipse-apoapsis/ort-server.git
+```
+
+## Run with Published Images
+
+To start the ORT Server with the required third-party services, run:
+
+```shell
+docker compose up
+```
+
+This will use the ORT Server images [published on GitHub](https://github.com/orgs/eclipse-apoapsis/packages?ecosystem=container).
+By default, the `main` tag is used.
+Those images are built from the `main` branch of the repository.
+To use a different tag, you can set the `ORT_SERVER_IMAGE_TAG` environment variable, for example:
+
+```shell
+ORT_SERVER_IMAGE_TAG=0.1.0-SNAPSHOT-001.sha.aa4d3fa docker compose up
```

Comment:
Didn't we swap out "SNAPSHOT" for "RC"? In general, I believe something that really exists should be used here, even if just an example.